nemo_text_processing/text_normalization/zh/data/money/currency.tsv

```diff
@@ -0,0 +1,21 @@
+￥	元
+$	美元
+USD	美元
+€	欧元
+EUR	欧元
+￡	英镑
+J￥	日元
+JPY￥	日元
+HK＄	港元
+HKD	港元
+฿	泰铢
+AUD	澳元
+A$	澳元
+SUR	卢布
+₽	卢布
+DEM	马克
+DM	马克
+FRF	法郎
+CAD	加元
+CAD$	加元
+FRF	法郎
```

Comment:
nit: is it possible to store currency codes (e.g. USD) and symbols (e.g. $) in separate files?

Reply:
It seems `currency.tsv` file only consider a bunch of currencies. My guess is that the selected currency set occurs most frequently in the Chinese datasets used here. Could you please share any thoughts? Given https://www.eurochange.co.uk/travel-money/world-currency-abbreviations-symbols-and-codes-travel-money, it seems many more currencies worth to add in this file.